edx_repo_tools/release/tag_release.py

```diff
@@ -773,9 +713,47 @@ def remove_ref_for_repos(repo_names, ref, session, use_tag=True):
     return modified
 
 
-def main():
-    parser = make_parser()
-    args = parser.parse_args()
+@click.command()
+@click.argument(
+    'ref', metavar="REF",
+)
+@click.option(
+    '--tag/--branch', "use_tag", is_flag=True,
+    help="Whether to create branches or tags in the repo"
```

Comment:
I can't tell from the click docs if this makes an option that must be provided. (BTW, it says is_flag=True is implied by the slash in the option name.)